python/lsst/ts/aos/utils/m1m3/m1m3_aav.py

```diff
@@ -182,9 +188,9 @@ def was_raised(self, start: pd.Timestamp, end: pd.Timestamp) -> bool:
         last_state = self.detailed_states[
```

Comment:
For the line above maybe you could give some valueError so that it's clear what is going on?

Reply:
Why? How? Detaild_states is designed to hold at least one state before start period, so iloc[-1] for that will work.